.github/workflows/tests.yml

```
@@ -82,16 +85,31 @@ jobs:
 
+  check_path:
+    # If steep is pull request then fetch patch and check if file '.github/workflows/test.yml'
+    # was changed. If it was changed then set 'check' to true.
+    # If steep is push then set 'check' to true.
+    name: Check if test.yml was changed
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check if test.yml was changed
+        id: check
+        run: echo ::set-output name=check::$(if [ -n "$(git diff --name-only HEAD^ HEAD | grep '.github/workflows/test.yml')" ]; then echo "true"; else echo "false"; fi)
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:31: Quote this to prevent word splitting

(shellcheck)